packages/jasmine/src/jasmine_runner.js

```diff
@@ -178,13 +189,21 @@ function main(args) {
       originalExecute();
     };
     // Special case!
-    // To allow us to test sharding, always run the specs in the order they are declared
+    // To allow us to test sharding, always run the specs in the order they are
+    // declared
     if (process.env['TEST_WORKSPACE'] === 'npm_bazel_jasmine' &&
         process.env['BAZEL_TARGET'] === '//test:sharding_test') {
       jrunner.randomizeTests(false);
     }
   }
 
+  if (seed) {
+    // Display that the seed has been overridden in bright green.
+    console.log(`\u001b[32mJasmine seed override: ${seed}\u001b[39m`);
```

Comment:
I'm not so sure about colors here, it looks bad in a terminal that doesn't support it. I think a lot of CLI tools detect whether the terminal environment appears to support color and only conditionally print it. To be consistent I'd just leave it out here - we should have a general approach for tools run under bazel